intel-retail/automated-self-checkout
# feat: Independent Docker image for low code (DLStreamer) version of the microservice

## PR Checklist



## What are you changing?

Creating minimum Docker image for the low code (DLStreamer) version of the pipeline.

## Issue this PR will close

close: #598 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- cd helm
- make build
- docker compose up
- connect to mqtt `inferenceEvnet` topic to see results

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
